applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -104,7 +104,8 @@ def ExecuteFinalizeSolutionStep(self):
 
     def ExecuteFinalize(self):
         if (self.model_part.GetCommunicator().MyPID() == 0):
-            self.output_file.close()
+            if (self.write_drag_output_file):
```

Comment:
I'd firstly check this, in that way you avoid the MPI check if it is not needed.

Reply:
Done.